### PR TITLE
BC-8514 - fix room permission check

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Short Description

## Links to Ticket and related Pull-Requests

BC-8514

## Checklist before merging

- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [ ] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [ ] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/pages/course-rooms/CourseRoomDetails.page.vue [hunk 1]
```diff
@@ -486,6 +491,11 @@ export default defineComponent({
 			this.courseId = courseId;
 
 			await this.courseRoomDetailsModule.fetchContent(courseId);
+
+			if (this.roomData.roomId) {
+				this.roomVariant = RoomVariant.COURSE_ROOM;
```
Comment (MartinSchuhmacher) [inline src/pages/course-rooms/CourseRoomDetails.page.vue:496]: if there is a roomId, shouldnt it be `RoomVariant.ROOM` ?

File: src/store/course-room-details.ts [hunk 2]
```diff
@@ -292,6 +297,7 @@ export default class CourseRoomDetailsModule extends VuexModule {
 			applicationErrorModule.resetError();
 			return;
 		}
+		console.log("error", error);
```
Comment (bergatco) [inline src/store/course-room-details.ts:300]: I think, this `console.log` should be removed.

File: src/pages/course-rooms/CourseRoomDetails.page.vue (hunk 1 above)
Comment (odalys-dataport) [reply src/pages/course-rooms/CourseRoomDetails.page.vue:496]: No, the naming of course-rooms is still "room-ish", so this is basically the `courseRoomId`

File: src/modules/page/room/RoomDetails.page.unit.ts [hunk 3]
```diff
@@ -318,7 +341,7 @@ describe("@pages/RoomsDetails.page.vue", () => {
 					const { wrapper } = setup({
 						isLoading: false,
 						roomVariant: RoomVariant.ROOM,
-						hasEditPermission: true,
+						permissions: ["course_edit"],
```
[the remark this answers is not in the document]
Comment (odalys-dataport) [reply src/modules/page/room/RoomDetails.page.unit.ts:344]: I asked myself the same question, just applied the prior code. I will take a closer look again

File: src/pages/course-rooms/CourseRoomDetails.page.vue (hunk 1 above)
Comment (MartinSchuhmacher) [reply src/pages/course-rooms/CourseRoomDetails.page.vue:496]: lets see when the time comes, when I won't be confused by that anymore... xD

**Review by MartinSchuhmacher — APPROVED** [review]

**Pull request merged**